app/src/free/java/de/danoeh/antennapod/dialog/RatingDialog.java

```diff
@@ -3,14 +3,15 @@
 import android.app.Dialog;
 import android.content.Context;
 import android.content.SharedPreferences;
+import android.util.Log;
+
 import androidx.annotation.Nullable;
 import androidx.annotation.VisibleForTesting;
-import android.util.Log;
+import androidx.appcompat.app.AlertDialog;
```

Comment:
Please don't touch unrelated files to keep the git history clean

Reply:
Agree, it became automatic thing for me.